### ISSUE TITLE
FrozenError: can't modify frozen String: "boolean"

### ISSUE THREAD

**Issue by n-rodriguez** [voormedia/rails-erd#381]
```sh
nicolas@tchoum-laptop:~/PROJECTS/CONCERTO/concerto$ bin/rake erd --trace
=> Checking config
=> Starting app (development)
** Invoke erd (first_time)
** Invoke erd:generate (first_time)
** Invoke erd:check_dependencies (first_time)
** Execute erd:check_dependencies
** Invoke erd:options (first_time)
** Execute erd:options
** Invoke erd:load_models (first_time)
** Execute erd:load_models
Loading application environment...
** Invoke environment (first_time)
** Execute environment
[2022-05-30T22:30:53.367701 #339193 #] DEBUG [] - ActiveRecord::Base:   SQL (0.7ms)  SELECT "regions"."id" AS t0_r0, "regions"."name" AS t0_r1, "regions"."created_at" AS t0_r2, "regions"."updated_at" AS t0_r3, "regions"."country_id" AS t0_r4, "regions"."map_identifier" AS t0_r5, "regions"."latitude" AS t0_r6, "regions"."longitude" AS t0_r7, "regions"."attachment_data" AS t0_r8, "region_translations"."id" AS t1_r0, "region_translations"."region_id" AS t1_r1, "region_translations"."locale" AS t1_r2, "region_translations"."created_at" AS t1_r3, "region_translations"."updated_at" AS t1_r4, "region_translations"."slug" AS t1_r5 FROM "regions" LEFT OUTER JOIN "region_translations" ON "region_translations"."region_id" = "regions"."id" ORDER BY "region_translations"."slug" ASC /*application:Concerto*/
[2022-05-30T22:30:53.368320 #339193 #] DEBUG [] - ActiveRecord::Base:   ↳ config/environment.rb:7:in `<main>'
[2022-05-30T22:30:53.390300 #339193 #] DEBUG [] - ActiveRecord::Base:   SQL (0.4ms)  SELECT "situations"."id" AS t0_r0, "situations"."created_at" AS t0_r1, "situations"."updated_at" AS t0_r2, "situations"."on_spanish_site" AS t0_r3, "situations"."attachment_data" AS t0_r4, "situation_translations"."id" AS t1_r0, "situation_translations"."situation_id" AS t1_r1, "situation_translations"."locale" AS t1_r2, "situation_translations"."created_at" AS t1_r3, "situation_translations"."updated_at" AS t1_r4, "situation_translations"."name" AS t1_r5, "situation_translations"."slug" AS t1_r6 FROM "situations" LEFT OUTER JOIN "situation_translations" ON "situation_translations"."situation_id" = "situations"."id" ORDER BY "situation_translations"."slug" ASC /*application:Concerto*/
[2022-05-30T22:30:53.390740 #339193 #] DEBUG [] - ActiveRecord::Base:   ↳ config/environment.rb:7:in `<main>'
Loading code in search of Active Record models...
** Execute erd:generate
Generating Entity-Relationship Diagram for 67 models...
Warning: Ignoring invalid model FriendlyId::Slug (table friendly_id_slugs does not exist)
Warning: Ignoring invalid association :connectors on Royce::Role (model Royce::Connector exists, but is not included in domain)
Warning: Ignoring invalid association :taggings on ActsAsTaggableOn::Tag (model ActsAsTaggableOn::Tagging exists, but is not included in domain)
Warning: Ignoring invalid association :taggings on Account (model ActsAsTaggableOn::Tagging exists, but is not included in domain)
Warning: Ignoring invalid association :tag_taggings on Account (model ActsAsTaggableOn::Tagging exists, but is not included in domain)
Warning: Ignoring invalid association :role_connectors on User (model Royce::Connector exists, but is not included in domain)
Warning: Ignoring invalid association :owned_taggings on User (model ActsAsTaggableOn::Tagging exists, but is not included in domain)
Warning: Ignoring invalid association :taggings on Account (model ActsAsTaggableOn::Tagging exists, but is not included in domain)
Warning: Ignoring invalid association :tag_taggings on Prospect (model ActsAsTaggableOn::Tagging exists, but is not included in domain)
Warning: Ignoring invalid association :taggings on Account (model ActsAsTaggableOn::Tagging exists, but is not included in domain)
Warning: Ignoring invalid association :tag_taggings on Customer (model ActsAsTaggableOn::Tagging exists, but is not included in domain)
Warning: Ignoring invalid association :role_connectors on User (model Royce::Connector exists, but is not included in domain)
Warning: Ignoring invalid association :owned_taggings on User (model ActsAsTaggableOn::Tagging exists, but is not included in domain)
rake aborted!
FrozenError: can't modify frozen String: "boolean"
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rails-erd-1.7.0/lib/rails_erd/domain/attribute.rb:124:in `block in type_description'
<internal:kernel>:90:in `tap'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rails-erd-1.7.0/lib/rails_erd/domain/attribute.rb:122:in `type_description'
(erb):9:in `block in entity_options'
(erb):8:in `each'
(erb):8:in `entity_options'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/3.1.0/erb.rb:905:in `eval'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/3.1.0/erb.rb:905:in `result'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rails-erd-1.7.0/lib/rails_erd/diagram/graphviz.rb:294:in `entity_options'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rails-erd-1.7.0/lib/rails_erd/diagram/graphviz.rb:230:in `block in <class:Graphviz>'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rails-erd-1.7.0/lib/rails_erd/diagram.rb:136:in `instance_exec'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rails-erd-1.7.0/lib/rails_erd/diagram.rb:136:in `block in generate'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rails-erd-1.7.0/lib/rails_erd/diagram.rb:135:in `each'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rails-erd-1.7.0/lib/rails_erd/diagram.rb:135:in `generate'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rails-erd-1.7.0/lib/rails_erd/diagram.rb:119:in `create'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rails-erd-1.7.0/lib/rails_erd/diagram.rb:74:in `create'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rails-erd-1.7.0/lib/rails_erd/tasks.rake:67:in `block (2 levels) in <main>'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/task.rb:281:in `block in execute'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/task.rb:281:in `each'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/task.rb:281:in `execute'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/sentry-ruby-core-5.3.1/lib/sentry/rake.rb:26:in `execute'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/task.rb:219:in `block in invoke_with_call_chain'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/task.rb:199:in `synchronize'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/task.rb:199:in `invoke_with_call_chain'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/task.rb:243:in `block in invoke_prerequisites'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/task.rb:241:in `each'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/task.rb:241:in `invoke_prerequisites'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/task.rb:218:in `block in invoke_with_call_chain'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/task.rb:199:in `synchronize'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/task.rb:199:in `invoke_with_call_chain'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/task.rb:188:in `invoke'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/application.rb:160:in `invoke_task'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/application.rb:116:in `block (2 levels) in top_level'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/application.rb:116:in `each'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/application.rb:116:in `block in top_level'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/application.rb:125:in `run_with_threads'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/application.rb:110:in `top_level'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/application.rb:83:in `block in run'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/application.rb:186:in `standard_exception_handling'
/home/nicolas/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/application.rb:80:in `run'
bin/rake:4:in `<main>'
Tasks: TOP => erd => erd:generate
```

**Comment by simi** [comment]
Any idea how to reproduce this on new Rails app?

**Comment by n-rodriguez** [comment]
> Any idea how to reproduce this on new Rails app?

I think this is due to `# frozen_string_literal: true` header : 

```ruby
# frozen_string_literal: true

class User < ApplicationRecord
end
```

**Comment by simi** [comment]
Seems rails-erd is not compatible with that. I can reproduce similar error with.

```ruby
RUBYOPT="--enable-frozen-string-literal" bundle exec rake test
```

**Comment by kerrizor** [comment]
Fixed in 1.7.1
